doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -71,6 +71,23 @@ Peripheral samples
   * The CLI command ``fem tx_power_control <tx_power_control>`` replaces ``fem tx_gain <tx_gain>`` .
     This change applies to the sample built with the :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
 
+
+Matter
+------
+
+  * With the inheritance of Zephyr's :ref:`zephyr:sysbuild` in the |NCS|, some changes are provided to the Matter samples and applications:
+
+    * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` Kconfig option is deprecated and you need to use the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA_BUILD` Kconfig option instead to enable or disable creating the factory data set during building a Matter sample.
+      To enable factory data support on your device, you still need to set the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA` to ``y``.
+    * Factory data output files are now located in the ``<application_name>/zephyr/`` directory within the build directory.
+    * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_MERGE_WITH_FIRMWARE` Kconfig option is deprecated in sysbuild and you need to use the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA_MERGE_WITH_FIRMWARE` Kconfig option instead to enable or disable merging the factory data HEX file with the final firmware HEX file.
+    * :kconfig:option:`SB_CONFIG_MATTER_OTA` Kconfig option has been added to enable or disable generating Matter OTA package during the building process.
+    * :kconfig:option:`CONFIG_CHIP_OTA_IMAGE_FILE_NAME` Kconfig option is deprecated and you need to use the :kconfig:option:`SB_CONFIG_MATTER_OTA_IMAGE_FILE_NAME` Kconfig option instead to define Matter OTA output filename.
+
+  .. note::
+    
+    If you want to build sample without using sysbuild you need to use the old Kconfig options.
```

Comment:
```suggestion
    If you want to build a sample without using sysbuild, you need to use the old Kconfig options.
```

Reply:
Not really sure why it's being documented how to build using child/parent at all, it is deprecated, it should not be used, it's included **ONLY** for customers to port their applications to sysbuild. If people want to use child/parent samples then they should stay using ncs 2.6

Reply:
Most likely, because we suspect that moving to sysbuild will be such a huge hit in the face for our customers that they would like to stay with 2.6.0, but they will have to use 2.7.0 to benefit from new Matter 1.3 revision. TBH majority of developers and QA engineers currently do not know how to move around in our SDK using new approach, so I would not expect it will be easy for our customers.

Reply:
It must not be deprecated. @LuDuda please comment on that.

Reply:
```
CMake Deprecation Warning at /tmp/bb/nrf/cmake/partition_manager.cmake:32 (message):
  

            ---------------------------------------------------------------------
            --- WARNING: Child and parent image functionality is deprecated   ---
            --- and should be replaced with sysbuild. Child and parent image  ---
            --- support remains only to allow existing customer applications  ---
            --- to build and allow porting to sysbuild, it is no longer       ---
            --- receiving updates or new features and it will not be possible ---
            --- to build using child/parent image at all in nRF Connect SDK   ---
            --- version 2.9 onwards.                                          ---
            ---------------------------------------------------------------------
```

Reply:
@nordicjm's assumption that customers do not migrate between revisions and must stay on NCS 2.6.0 is new to me, and it certainly does not reflect reality where our customers do migrate (e.g., for new Matter/Sidewalk/BLE versions, etc.). We see it also in customer surveys.

In NCS 2.7.0 and NCS 2.8.0, we need to enable our customers to use child images for their legacy out-of-tree applications. We have outlined a Matter-specific migration guide here, as we have eliminated the need to set the KConfig ROOT as previously requested. Therefore, this will require a manual migration by the customer.

Regarding the migration guide to sysbuild - we still need documentation here - so customers know how to do it. Ideally 
 I would love to have it even before the migration begins at Nordic. :)

Btw. Deprecation means that the feature will be soon removed and customers should not use it for new projects. But can keep it until is removed (3 releases by Nordic policy).

Reply:
Building using child-parent images is deprecated. Any references to this method should be clearly marked as such. The remark should point users to the sysbuild migration guide. Existing documentation referring to this build method should be kept until the deprecation period is over and support for this method is removed from the build system.